src/main/webapp/i18n/en/quizExercise.json

```diff
@@ -226,7 +226,17 @@
                 "title": "Short Quiz Title",
                 "questionTitle": "Question Title",
                 "shortQuestionTitle": "Short Question Title"
-            }
+            },
+            "matchLetterCase": {
+                "title": "Match Letter Case",
+                "explanation": "Select if the letter case of the answers provided matters in the evaluation (Example: should 'Answer' and 'answer' be evaluated as the same?)"
+            },
+            "matchPercentage": {
+                "title": "Match Answers Exactly",
+                "explanationTitle": "Similarity&nbsp;%",
```

Comment:
```suggestion
                "explanationTitle": "Similarity",
```

Reply:
![Uploading image.png…]()

Reply:
This was added explicitly by @jpbernius before ;-)

Reply:
oh well, then we should remove it

Reply:
Wait so the % was already there. What I tried to do is have a non-breaking space.
Apparently I failed miserably.